extensions/wikia/AdEngine/js/AdInfoTrackerHelper.js

```diff
@@ -68,6 +68,9 @@ define('ext.wikia.adEngine.adInfoTrackerHelper',  [
 			'bidder_10': transformBidderPrice('appnexusAst', realSlotPrices, slotPricesIgnoringTimeout),
 			'product_chosen': '',
 			'product_lineitem_id': slotFirstChildData.gptLineItemId || '',
+			'creative_id': slotFirstChildData.gptCreativeId || '',
+			'creative_size': slotFirstChildData.gptCreativeSize || '',
```

Comment:
What do you think about parsing `[728,90]` to `728x90`? Keep in mind it may be `null` (and maybe other values?)